astroid/brain/brain_hypothesis.py

```diff
@@ -0,0 +1,54 @@
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/master/COPYING.LESSER
+"""
+Astroid hook for the Hypothesis library.
+
+Without this hook pylint reports no-value-for-parameter for use of strategies
+defined using the `@hypothesis.strategies.composite` decorator.  For example:
+
+    from hypothesis import strategies as st
+
+    @st.composite
+    def a_strategy(draw):
+        return draw(st.integers())
+
+    a_strategy()
+
+"""
+
+import astroid
+
+COMPOSITE_NAMES = (
```

Comment:
This doesn't seem to detect when people are importing `composite` directly rather than referencing it via a module import. I bet a lot of people are doing that. Is there any way to test for that case?

Reply:
Yep... I just have to add the bare name to the list of decorators we treat.  I think this is OK since I've added "is the first argument named `draw`" to the detection heuristic.